### PR TITLE
Make min unwind consistent with unwind

### PR DESCRIPTION
**Pull request by thk123**
- [x] Each commit message has a non-empty body, explaining why the change was made.
- [x] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [x] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [na] My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [x] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

Ideally there should be documentation for the incremental feature in the docs, but it doesn't seem to be present. @peterschrammel are you aware of any that can be imported in?

### CODE REVIEW

File: src/goto-checker/symex_bmc_incremental_one_loop.cpp [hunk 1]
```diff
@@ -10,6 +10,7 @@ Author: Peter Schrammel, Daniel Kroening, kroening@kroening.com
 
 #include <util/source_location.h>
 #include <util/xml.h>
+#include <iostream>
```
Comment (thk123) [inline src/goto-checker/symex_bmc_incremental_one_loop.cpp]: :no_entry_sign: Oops

Comment (thk123) [reply src/goto-checker/symex_bmc_incremental_one_loop.cpp]:
```suggestion
```

File: regression/cbmc-incr-oneloop/ignore-before-unwind/include_first_unwind0.desc [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+main.c
+--ignore-properties-before-unwind-min --incremental-loop main.0 --unwind-max 2 --unwind-min 0
+^EXIT=10$
+^SIGNAL=0$
+^\[main.assertion.\d+\] line \d+ property: FAILURE$
+^VERIFICATION FAILED$
+--
+^warning: ignoring
+--
+This test correctly fails because the first iteration of the loop violates the
```
Comment (thk123) [inline regression/cbmc-incr-oneloop/ignore-before-unwind/include_first_unwind0.desc:11]: This copy pasted comment could be elaborated

File: regression/cbmc-incr-oneloop/ignore-before-unwind/include_first_unwind0.desc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+CORE
```
Comment (thk123) [inline regression/cbmc-incr-oneloop/ignore-before-unwind/include_first_unwind0.desc:1]: This commit message summary is too long for GH:

>  Update unwind-min tests to check the precise behaviour for different …

File: src/goto-checker/bmc_util.h [hunk 4]
```diff
@@ -221,6 +221,11 @@ void run_property_decider(
   "                              of loop L\n" \
   "                              (use --show-loops to get the loop IDs)\n" \
   " --unwind-min nr              start incremental-loop after nr unwindings\n" \
+  "                              but before solving that iteration. If for \n" \
+  "                              example it is 1, then the loop will be \n" \
+  "                              unwound once, and immediately checked. \n" \
+  "                              Note: this means min-unwind 1 or \n"\
+  "                              0 all properties are checked. \n" \
```
Comment (thk123) [inline src/goto-checker/bmc_util.h]: Remove spaces after \n
```suggestion
  "                              but before solving that iteration. If for\n" \
  "                              example it is 1, then the loop will be\n" \
  "                              unwound once, and immediately checked.\n" \
  "                              Note: this means min-unwind 1 or\n"\
  "                              0 all properties are checked.\n" \
```

File: src/goto-checker/symex_bmc_incremental_one_loop.cpp (hunk 1 above)
Comment (peterschrammel) [inline src/goto-checker/symex_bmc_incremental_one_loop.cpp]: spurious change

File: src/goto-checker/bmc_util.h [hunk 5]
```diff
@@ -221,6 +221,11 @@ void run_property_decider(
   "                              of loop L\n" \
   "                              (use --show-loops to get the loop IDs)\n" \
   " --unwind-min nr              start incremental-loop after nr unwindings\n" \
+  "                              but before solving that iteration. If for \n" \
+  "                              example it is 1, then the loop will be \n" \
+  "                              unwound once, and immediately checked. \n" \
+  "                              Note: this means min-unwind 1 or \n"\
```
Comment (peterschrammel) [inline src/goto-checker/bmc_util.h]:
```suggestion
  "                              Note: this means for min-unwind 1 or \n"\
```
?

**Comment by thk123** [comment]
@peterschrammel ready for rereview

**Review by peterschrammel — APPROVED** [review]

**Review by NlightNFotis — APPROVED** [review]

**Pull request merged**